### DISCUSSION TITLE
AAX not showing in pro tools 12

### DISCUSSION THREAD

**Discussion by 4pf5** [apohl79/audiogridder#1198] [Help]
I installed both the plugin and server downloads but when I open pro tools 12 on monterey mac studio m1, the aax does not appear in the plugin drop down for pro tools 12. 

When I use bluecat audio patchwork to open the vst it says the file is in 32 bit.

Can you send me the 64 bit aax?

**Comment by apohl79** [comment]
All binaries are 64bit. You are probably using the wrong CPU architecture. If PT runs under rosetta you need the intel build. If not you need the arm build. The universal build works for both.
